tools/python/mwm_downloader.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import re
+import time
+import sys
+import urllib.request
+import logging
+
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+
+DIRECT_MAP_URL = 'http://direct.mapswithme.com/direct/'
+ROOT = os.path.dirname(os.path.realpath(__file__))
+DEFAULT_DOWNLOAD_DIRECTORY = os.path.join(os.path.dirname(os.path.dirname(ROOT)), 'data')
+MAP_REVISION_REGEXP = re.compile(r'(\d{6})')
+MWM_NAME_REGEXP = re.compile(r'"(\S+).mwm"')
+
+logger = logging.getLogger(__name__)
+
+
+def parse_html_by_regexp(url, regexp):
+    parse_result = []
+    with urllib.request.urlopen(url) as f:
+        lines = f.read().decode('utf-8').split('\n')
+        for line in lines:
+            match = regexp.search(line)
+            if match:
+                parse_result.append(match.group(1))
+    return parse_result
+
+
+def get_possible_mwm_revisions():
+    return [int(revision) for revision in parse_html_by_regexp(DIRECT_MAP_URL, MAP_REVISION_REGEXP)]
```

Comment:
Again, IMHO, but map(int, smth) just easier to read

Reply:
Thanks, fixed.